docker/ansible-in-docker/build.bash

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+set -vx
+
+function buildAnsible {
+    dockerTag=oso-ansible:${ansibleVersion}
+
+    bashScript=./oso-ansible-playbook-${ansibleVersion}
+
+    docker build --tag ${dockerTag} - << EndOfDockerfile
+FROM fedora:25
+
+RUN yum -y update \
+ && yum -y install ${ansibleRPM} \
+ && yum clean all
+EndOfDockerfile
+
+    cat > ${bashScript} << EndOfBashScript
+#!/bin/bash
+#set -vx
```

Comment:
Add deprecation section to alert the user when the version of ansible being used has been deprecated.